course-materials/labs/week-1-lab.Rmd

```diff
@@ -24,12 +24,13 @@ library(tidyverse)
 library(readr)
 library(gt)
 library(openintro)
+library(here)
 
 # Where is the root directory where all your labs live? If you want to set up as an .RProj, go for it. 
-rootdir <- "~/Dropbox/Teaching/UCSB/EDS_222/EDS222_data/"
+rootdir <- "root_directory_labs"
```

Comment:
I'll go through future labs and remove the Dropbox file paths. I did that for the slides but the labs escaped me. Thanks, Leo!